examples/system/applications/console/src/main.rs

```diff
@@ -0,0 +1,350 @@
+
+            let resp = context
+                .storage_caller
+                .blocking_call(&Request::AppendKey(key, value))
+                .expect("Failed to perform a blocking_call");
+
+            print_resp(context, &resp);
+        }
+    }
+
+    pub mod get {
+        use super::*;
+
+        pub const HELP: &str = "Retrieves the value for the given key from storage.
+
+  Example:
+  get my-key";
+
+        pub fn cmd(
+            _menu: &Menu<Context>,
+            item: &Item<Context>,
+            args: &[&str],
+            context: &mut Context,
+        ) {
+            let key = Key::from(menu::argument_finder(item, args, "key").unwrap().unwrap());
+
+            log::debug!("[console] Get storage value for key='{}'", key);
+
+            let resp = context
+                .storage_caller
+                .blocking_call(&Request::Get(key))
+                .expect("Failed to perform a blocking_call");
+
+            print_resp(context, &resp);
+        }
+    }
+
+    pub mod invalidate {
+        use super::*;
+
+        pub const HELP: &str = "Invalidates the key in storage.
+
+  Example:
+  invalidate my-key";
+
+        pub fn cmd(
+            _menu: &Menu<Context>,
+            item: &Item<Context>,
+            args: &[&str],
+            context: &mut Context,
+        ) {
+            let key = Key::from(menu::argument_finder(item, args, "key").unwrap().unwrap());
+
+            log::debug!("[console] Invalidate storage key='{}'", key);
+
+            let resp = context
+                .storage_caller
+                .blocking_call(&Request::InvalidateKey(key))
+                .expect("Failed to perform a blocking_call");
+
+            print_resp(context, &resp);
+        }
+    }
+
+    pub mod gc {
+        use super::*;
+
+        pub const HELP: &str = "Perform a garbage collection on storage.
+
+  Example:
+  gc";
+
+        pub fn cmd(
+            _menu: &Menu<Context>,
+            _item: &Item<Context>,
+            _args: &[&str],
+            context: &mut Context,
+        ) {
+            log::debug!("[console] Garbage collect storage");
+
+            let resp = context
+                .storage_caller
+                .blocking_call(&Request::GarbageCollect)
+                .expect("Failed to perform a blocking_call");
+
+            print_resp(context, &resp);
+        }
+    }
+}
+
+mod net {
+    use super::*;
+
+    pub mod sendto {
+        use super::*;
+
+        pub const HELP: &str = "Send a UDP message.
+  
+    Example:
+    sendto 192.0.2.2 4567 hello";
+
+        pub fn cmd(
+            _menu: &Menu<Context>,
+            item: &Item<Context>,
+            args: &[&str],
+            context: &mut Context,
+        ) {
+            let addr = menu::argument_finder(item, args, "addr").unwrap().unwrap();
+            let mut addr_octets = [0_u8; 4];
+            for (idx, part) in addr.split('.').into_iter().enumerate() {
+                addr_octets[idx] = part.parse().unwrap();
```

Comment:
I suppose the fact that invalid input will cause the system to crash is acceptable for a simple example.

Reply:
yeah, no application level error handling is done, and it's kinda fun to crash the app and see the rest of the system remain operational.